README.md

```diff
@@ -13,6 +13,11 @@ The keys will generated automaticly and will be available after login to your ac
 - for weatherapi: https://www.weatherapi.com/docs/#intro-getting-started
 - for rapidapi: https://docs.rapidapi.com/docs/keys-and-key-rotation#how-to-find-your-api-key
 
+Create better_weather_keys.txt file that will store your api keys generated in previous steps.
+The file should contain the name of the website followed by the appropriate API key separated by a comma, e.g:
+wheatherapi.com,9b27197b46d...b142443242202
+meteostat.net,5cc4a3480mshd9...p111424jsn9e365e801bf8
```

Comment:
Kudos for how this message is crafted - short, and to the point. I see progress in this area👏

Reply:
Gracias.